docs/include/conf/portland/visiting.rst

```diff
@@ -41,7 +41,7 @@ The MAX
 
 The `MAX <http://trimet.org/max>`__ is Portland's major light-rail system. The MAX primarily connects Portland with the surrounding cities, such as Gresham, Beaverton, Clackamas, and Hillsboro.
 
-If you're flying into Portland Airport (PDX), we recommend taking the MAX into downtown Portland (service to downtown Portland runs between 5 am and midnight). There is a MAX station for the `Red Line <http://trimet.org/schedules/maxredline.htm>`__ in the airport itself.
+If you're flying into Portland Airport (PDX), we recommend taking the MAX into downtown Portland (service to downtown Portland runs between 5 am and midnight). There is a MAX station for the `Red Line <http://trimet.org/schedules/maxredline.htm>`__ in the airport itself. You may purchase MAX tickets from a ticket machine at the airport. Be sure to buy a ticket before boarding the train.
```

Comment:
I'd say you *can* purchse tickets here. And I'd also skip "Be sure to buy a ticket before boarding the train." but that might be personal preference.

Reply:
As you can't buy a ticket on the train, I think the warning is OK. (The MAX police do check once in a while.)